src/NUnitFramework/framework/Constraints/UniqueItemsConstraint.cs

```diff
@@ -174,5 +192,39 @@ private Type GetGenericTypeArgument(IEnumerable actual)
 
             return null;
         }
+
+        private sealed class CaseInsensitiveCharComparer : IEqualityComparer<char>
+        {
+            public bool Equals(char x, char y)
+            {
+                return char.ToLower(x) == char.ToLower(y);
+            }
+
+            public int GetHashCode(char obj)
+            {
+                return char.ToLower(obj).GetHashCode();
+            }
+        }
+
+        internal sealed class UniqueItemsContstraintResult : ConstraintResult
```

Comment:
```suggestion
        internal sealed class UniqueItemsConstraintResult : ConstraintResult
```

Small typo. 🙂 Good thinking making it internal sealed though!

Reply:
thanks!